pkg/clustermesh/kvstoremesh/remote_cluster.go

```diff
@@ -317,20 +317,17 @@ func (o *syncer) OnSync(ctx context.Context) {
 	}
 }
 
-func newReflector(local kvstore.BackendOperations, cluster, prefix, suffix string, factory store.Factory, synced *resources) reflector {
+func newReflector(logger *slog.Logger, local kvstore.BackendOperations, cluster, prefix, suffix string, factory store.Factory, synced *resources) reflector {
```

Comment:
Nit. This change and the ones below could be reverted (apparently we don't have a linter that complains about unused arguments).